examples/moodole_autorunonce

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+
+hostname='ole'
+hotspotname='treehouse'
+moodleversion='3.4'
```

Comment:
moodole ;)

Reply:
Yeah, I was not sure about how to make this part less confusing. `moodle` and `moodoleI` are pretty close. Here I'm refering to the actual moodle version 3.4. Maybe we should just leave `3.4` in the url at this time...

@mappuji Are we having separate directory (`3.4`, `3.5`...) for any new version of moodle in the future?

Reply:
yes we will have a new version if there is new coming 😄